### ISSUE TITLE
Configure Plugin to only send Publish messages.

### ISSUE THREAD

**Issue by Schweig** [emqx/emqx-web-hook#155]
As it says in the title I want to configure the plugin to only send requests when publish messages are sent. My config file looks as follows but I still receive the connected and disconnected requests 

`web.hook.api.url = http://127.0.0.1:8080

web.hook.rule.client.connected.1     =
web.hook.rule.client.disconnected.1  =
web.hook.rule.client.subscribe.1     =
web.hook.rule.client.unsubscribe.1   =
web.hook.rule.session.created.1      =
web.hook.rule.session.subscribed.1   =
web.hook.rule.session.unsubscribed.1 =
web.hook.rule.session.terminated.1   =
web.hook.rule.message.publish.1      = {"action": "on_message_publish"}
web.hook.rule.message.deliver.1    =
web.hook.rule.message.acked.1        =
`

**Comment by HJianBo** [comment]
Have you tried to comment on these lines? like this:

```bash
##web.hook.rule.session.terminated.1   =
web.hook.rule.message.publish.1      = {"action": "on_message_publish"}
##web.hook.rule.message.deliver.1    =
```

**Comment by Schweig** [comment]
Ok do I need to restart the whole broker to get the changes or should i just restart the plugin

**Comment by HJianBo** [comment]
Yes, you should reload the plugin:
```
emqx_ctl plugins reload emqx_web_hook
```

**Comment by xlb0479** [comment]
> Have you tried to comment on these lines? like this:
> 
> ```shell
> ##web.hook.rule.session.terminated.1   =
> web.hook.rule.message.publish.1      = {"action": "on_message_publish"}
> ##web.hook.rule.message.deliver.1    =
> ```

How to do this by docker env?
I use emqx-docker and how to comment out plugin variable by enviroment?

**Comment by zhanghongtong** [comment]
@xlb0479 EMQ X Docker Image does not currently support configuration via environment variable annotations, you can manually mount this file into the container from the outside

**Comment by daadu** [comment]
This plugin's default behaviour should be to disable all actions. comment out the config. anyone who need it can uncomment.

This will be a lot easier for docker user.

**Comment by zhanghongtong** [comment]
@HJianBo  Can we comment out the default configuration in `emqx_web_hook.conf` ？Users can open it if they want

**Comment by daadu** [comment]
This is a trivial solution. @HJianBo Let us know if you agree with it.

**Comment by HJianBo** [comment]
Yes, it is.  @daadu  @zhanghongtong 

It will be released at v4.1.1. Thank you for your suggestion  :-)
